frontend/src/components/Hierarchymap.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import HierarchyMap from './Hierarchymap';
+
+export default {
+  title: 'HierarchyMap',
+  component: HierarchyMap,
+};
+
+const exampleOccupation = {
+  name: 'Software Developer',
+  level: 2,
+  mapHierarchy: {
+    technicalLevelName: 'Intermediate',
+    pathwayName: 'Development',
+  },
+  overview: 'Develops and maintains software applications.',
+  summary: '<p>In-depth details about the role of a software developer...</p>',
+};
+
+const Template = (args) => <HierarchyMap {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  occupation: exampleOccupation,
+  routeName: 'Digital',
+};
```

Comment:
Let's re-write this bit following the same setup we use in the other stories files in our code - your component doesn't take any props, so there's no reason for a template that spreads arguments